### ISSUE TITLE
Why didn't you start from the latest Naotoshi Seo commit?

### ISSUE THREAD

**Issue by db4** [JoakimSoderberg/imageclipper#11]
I also made a private fork of imageclipper some time ago and now looking if it's possible to rebase it on top of your fork and contribute some code. But it turned out that you did not use the latest sources from https://code.google.com/p/imageclipper (so some functionality is lost) and did not preserve the history. Why didn't you just import http://imageclipper.googlecode.com/svn/trunk into git using `git svn` before publishing on github?

**Comment by JoakimSoderberg** [comment]
Hrm, yea that is stupid of me... I thought I had done that, but I see now that I did not, and I don't remember the reason now :/

But kind of late to fix that now, so if you can manage to rebase that would be good

**Comment by db4** [comment]
Ok, but how to integrate all legacy Naotoshi code into your repo? I tend to think that rebasing your commits on top of Naotoshi Seo's ones and rewriting the history would be better option (preserving chronological order of changes etc.) but it can be really too much trouble for others

**Comment by JoakimSoderberg** [comment]
Yes too many people have forked this repo for it to be worth it... Where is the repo you imported... I could try to apply the natoshi code.

**Comment by db4** [comment]
I've just put it to github: https://github.com/db4/imageclipper
